src/main/java/edu/usf/cutr/grha/gtfs/GtfsConstants.java

```diff
@@ -0,0 +1,16 @@
+package edu.usf.cutr.grha.gtfs;
+
+public class GtfsConstants {
+
+    public static final String FAKE_AGENCY_ID = "CUTR";
+    public static final String FAKE_AGENCY_NAME = "Chicago x Tampa";
+    public static final String FAKE_AGENCY_URL = "www.fake.url.edu";
+    public static final String AGENCY_TIMEZONE = "EST";
```

Comment:
GTFS spec says:

>Timezone - TZ timezone from the https://www.iana.org/time-zones. Timezone names never contain the space character but may contain an underscore. Refer to http://en.wikipedia.org/wiki/List_of_tz_zones for a list of valid values.
Example: Asia/Tokyo, America/Los_Angeles or Africa/Cairo.

So timezone should be `America/Chicago`.